POH-README.md

```diff
@@ -1,24 +1,48 @@
-# Portable Open Hospital - POH
+# ![](./oh.ico) OH - Open Hospital Portable | Client 
 
-Open Hospital (https://www.open-hospital.org/en/) is a free and open source
-software for healthcare data management. Its portable version (Portable Open
-Hospital, POH) allows you to take along with you Open Hospital and run it on
-any computer, keeping the data you have saved. With POH, we also hope to
-reach the goal to make Open Hospital easily installable so that even somebody
-with no experience of Java or MySQL can try or use it.
+OH - Open Hospital (https://www.open-hospital.org/en/) is a free and open source
+software for healthcare data management. Its portable version allows you to take
+along with you Open Hospital and run it on any computer, keeping the data you have saved.
+With this distribution we also hope to reach the goal to make Open Hospital easily installable
+so that even somebody with no experience of Java or MySQL can try or use it.
 
-POH allows to use Open Hospital on a computer, easily move the installation on
+Open Hospital can run on any computer and it is easily possible to move the installation on
 another computer or even run it from a USB stick or drive. All you have to do
-is to copy the root installation directory of POH to your favourite path, where
-the program and the data will be kept. POH uses its own version of the Java Virtual
+is to copy the root installation directory of OH to your favourite path, where
+the program and the data will be kept. OH uses its own version of the Java Virtual
 Machine and the MySQL server and everything is contained in the root
-installation directory. POH is released under the GNU GPL 3.0 License.
+installation directory. OH is released under the GNU GPL 3.0 License.
 
 The Linux version has been tested on different distributions and versions,
-including Ubuntu 16.04 i386 (32bit) and up to Ubuntu 20.04 x64 (64bit).
+including Ubuntu 16.04 i386 (32bit) and up to Ubuntu 21.04 x64 (64bit).
 The Windows version has been tested on Windows 10.
 
-## Running POH - Quickstart
+**This repo is experimental and is used to test the latest Open Hospital releases and features. Use at your own risk !**
+
+# Running OH - Ultra-quickstart
+
+**on Linux:**
+
+To download and launch the Open Hospital package contained in this distribution open a shell and type (or copy and paste):
+
+```
+bash <(wget -qO- https://raw.githubusercontent.com/mizzioisf/openhospital-client/main/go.sh)
+```
+
+# Running OH - Quickstart
+
+**Common to all Operating Systems / architectures:**
+
+*(see OH downloads on sourceforce for full release packages)*
+
+- clone the repository with git:
+```
+git clone https://github.com/mizzioisf/openhospital-client
+```
+- browse to the directory:
+```
+cd openhospital-client
+```

```

Comment:
This file will go in the already bundle distribution, make sense to suggest to clone something from another non-official repository?

Reply:
I suggest to remove this section and remain with the title only `# Running POH - Quickstart`